kafka-broker/kafka-broker-install.sh

```diff
@@ -2,13 +2,20 @@
 
 ## Script to install a kafka & zookeeper broker
 
-# Install Java
 sudo yum update -y
 sudo yum install -y java-1.8.0-openjdk-devel
 
 # Install Amazon SSM agent
 sudo yum install -y https://s3.amazonaws.com/ec2-downloads-windows/SSMAgent/latest/linux_amd64/amazon-ssm-agent.rpm
 
+# Install python-pip
+# pip is not available in CentOS 7 core repositories there is a requirement to enable EPEL repositories prior
+sudo yum --enablerepo=extras install epel-release
+sudo yum install -y python-pip
+sudo yum install -y gcc
+sudo yum install -y python-devel
+sudo pip install -y https://github.com/dwp/acm-pca-cert-generator/releases/download/0.1.0/acm_pca_cert_generator-0.1.0.tar.gz
+
```

Comment:
The purist in me would like for this AMI to not have dev tools on it once those pip dependencies have been compiled. Can you do this at this point please?:

`sudo yum remove -y gcc python-devel`

That said, we've already got Java and Python on the box anyway so...